src/views/index/home.njk

```diff
@@ -6,11 +6,21 @@
 {% block main_content %}
   <form action="" method="POST">
     <h1 class="govuk-heading-l">{{ i18n.startPageTitle }}</h1>
+
     <p class="govuk-body">{{ i18n.useThisService }}</p>
-    <div class="govuk-inset-text">
-      {{ i18n.guidePageVerificationServiceNote }}
+    <p class="govuk-body">{{ i18n.guidePageVerificationServiceNote }}
       <a href="https://www.gov.uk/guidance/companies-house-identity-verification-standard" id="identity-verification-standard-link">{{ i18n.linkGuidePageVerificationService }}</a>.
+    </p>
+    <p class="govuk-body">{{ i18n.keepRecordsOfTheIdentityChecks }}</p>
+
+    <div class="govuk-warning-text">
+      <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+      <strong class="govuk-warning-text__text">
+        <span class="govuk-visually-hidden">Warning</span>
```

Comment:
As above

Reply:
Added text to common.json locales file to account for Welsh translation